vcd/datasource_vcd_edgegateway.go

```diff
@@ -97,6 +107,90 @@ func datasourceVcdEdgeGateway() *schema.Resource {
 				Computed:    true,
 				Description: "'accept' or 'deny'. Default 'deny'",
 			},
+			"fips_mode_enabled": &schema.Schema{
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "Enable FIPS mode. FIPS mode turns on the cipher suites that comply with FIPS. (False by default)",
+			},
+			"use_default_route_for_dns_relay": &schema.Schema{
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "If true, default gateway will be used for the edge gateways' default routing and DNS forwarding.(False by default)",
+			},
+			"external_network": {
+				Type:        schema.TypeSet,
+				Description: "One or more blocks with external network information to be attached to this gateway's interface",
+				Computed:    true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": {
+							Computed:    true,
+							Type:        schema.TypeString,
+							Description: "External network name",
+						},
+						"enable_rate_limit": {
+							Computed:    true,
+							Type:        schema.TypeBool,
+							Description: "Enable rate limitting",
+						},
+						"incoming_rate_limit": {
+							Computed:    true,
+							Type:        schema.TypeFloat,
+							Description: "Incoming rate limit (Mbps)",
```

Comment:
Please confirm that Mbps is what API is using (just want to make sure we avoid nasty conversion issues).

Reply:
Yes. And funny enough it works properly with float values (included in tests as well):
<img width="803" alt="image" src="https://user-images.githubusercontent.com/15804230/69731696-04d08480-1133-11ea-96fa-51a90e9fb31c.png">

Reply:
We have case where API and UI don't match Types.

Reply:
Yep, but these numbers in screenshot were created using provider, not entered manually.